Document basic management of community projects

matthiasr opened the pull request:
Since this repository has a wider scope, move the Prometheus & The Ecosystem Community Meeting document to a separate file.

Add basic documentation on the practices around community projects.

I was trying to figure out how to handle #43 but had to ask @SuperQ.

matthiasr committed:
Document basic management of community projects

Since this repository has a wider scope, move the Prometheus & The
Ecosystem Community Meeting document to a separate file.

Add basic documentation on the practices around community projects.

Signed-off-by: Matthias Rampke <matthias@prometheus.io>

SuperQ commented on REPOSITORIES.md:
```diff
@@ -0,0 +1,32 @@
+
+1. Comment on the issue that you are willing to sponsor this repository.
+1. Create a [team](https://github.com/orgs/prometheus-community/teams) and add the maintainers for the new repository, unless there is already an appropriate team.
+1. Create the repository.
```
When adopting an existing project, we don't create the repo. It's transfered as-is from the original owner. Unless we're doing a hard fork.

SuperQ reviewed (approved):
LGTM.

matthiasr committed:
Swap order of create/import instructions

We move existing projects more often than we create new ones.
Additinoally, it's more obvious from the instructions that moving
doesn't work if you want to create a new project, so it's harder to do
the wrong thing when following the document top-to-bottom.

Signed-off-by: Matthias Rampke <matthias@prometheus.io>
